kyuubi-rest-client/src/main/java/org/apache/kyuubi/client/KyuubiRestClient.java

```diff
@@ -198,16 +198,13 @@ public KyuubiRestClient build() {
         throw new IllegalArgumentException("hostUrl cannot be blank.");
       }
 
-      if (authSchema == AuthSchema.SPNEGO && StringUtils.isBlank(spnegoHost)) {
+      if (authHeaderMethod == AuthHeaderMethod.SPNEGO.SPNEGO && StringUtils.isBlank(spnegoHost)) {
```

Comment:
@turboFei seems `SPNEGO` is repeated

Reply:
good catch, raise https://github.com/apache/incubator-kyuubi/pull/2843 to followup